docs/src/pages/getting-started/page-layout-examples/checkout/Checkout.js

```diff
@@ -20,34 +20,34 @@ const styles = theme => ({
   },
   layout: {
     width: 'auto',
-    marginLeft: theme.spacing.unit * 2,
-    marginRight: theme.spacing.unit * 2,
-    [theme.breakpoints.up(600 + theme.spacing.unit * 2 * 2)]: {
+    marginLeft: theme.spacing(2),
+    marginRight: theme.spacing(2),
+    [theme.breakpoints.up(600 + theme.spacing(2) * 2)]: {
```

Comment:
Same again

Reply:
Changed my mind. I think that it's fine like this. It's about `theme.spacing(2) + theme.spacing(2)`.